src/tests/ftest/scrubber/frequency.yaml

```diff
@@ -0,0 +1,67 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+timeout: 1800
+setup:
+  start_servers_once: false
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31516
+      log_file: daos_server1.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+  transport_config:
+    allow_insecure: true
+agent_config:
+  transport_config:
+    allow_insecure: true
+dmg:
+  transport_config:
+    allow_insecure: true
+pool:
+  scm_size: 6000000000
+  nvme_size: 54000000000
+  svcn: 4
+  control_method: dmg
```

Comment:
```suggestion
```

Reply:
Yes. This is the problem when you start the initial template with cut and paste :-)